src/components/Create/components/Wizard/Page.tsx

```diff
@@ -0,0 +1,90 @@
+import { ArrowLeftOutlined, ArrowRightOutlined } from '@ant-design/icons'
+import { Button, ButtonProps, Space } from 'antd'
+import { usePage } from './hooks'
+
+const BackButton = (props: ButtonProps) => {
+  return (
+    <Button size="large" {...props}>
```

Comment:
**suggestion**: perhaps we can use the `icon` prop for consistency.

Reply:
I think it is better to be consistent with our components here.

For the BackButton we want it showing on the left (which icon= in antd will give us) but NextButton, we want it on the right (antd will not give us this